pkg/cmd/pipeline/start.go

```diff
@@ -167,7 +182,7 @@ like cat,foo,bar
 	)
 
 	c.Flags().StringSliceVarP(&opt.Labels, "labels", "l", []string{}, "pass labels as label=value.")
-	c.Flags().StringArrayVarP(&opt.Workspaces, "workspace", "w", []string{}, "pass the workspace.")
```

Comment:
We can keep it till

```
pass one or more workspaces to map to the corresponding physical volumes
```

We have provided the details about how to pass in help section, this may be confusing if it is limited to these two

Reply:
addressed :+1: